### PR TITLE
VZ-6151: Runbook for invalid shape specified to Load Balancer  

### PR DESCRIPTION
**Pull request by bhatpmk**
Adds a runbook for the use case when the Verrazzano custom resource specifies a shape for the OCI Load Balancer, which is invalid. 
As part of this change, removed the files which are not captured by k8s-dump-cluster.sh, from VerrazzanoAnalysisTool.md

### CODE REVIEW

File: content/en/docs/troubleshooting/diagnostictools/analysisadvice/IngressInvalidShape.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+### Summary
+Analysis detected that the Verrazzano installation failed while installing the NGINX Ingress Controller.
+
+The root cause appears to be that Verrazzano custom resource provided an invalid shape for Oracle Cloud Infrastructure Load Balancer.
+
+### Steps
+1. Review the messages from the supporting details for the allowed shape for Oracle Cloud Infrastructure Load Balancer.
+2. Refer to the Oracle Cloud Infrastructure documentation related to managing [Load Balancer](https://docs.oracle.com/en-us/iaas/Content/Balance/Tasks/managingloadbalancer.htm#console).
```
Comment (rosemarymarano) [inline content/en/docs/troubleshooting/diagnostictools/analysisadvice/IngressInvalidShape.md:16]: Step 2 directs users to the same information as in the first link in Related Information. I suggest that you either remove Step 2 here OR remove the same link from Related Information.
Also, if you decide to keep Step 2, I suggest this wording, "Refer to the Oracle Cloud Infrastructure documentation, [Load Balancer Management](https://docs.oracle.com/en-us/iaas/Content/Balance/Tasks/managingloadbalancer.htm#console).

**Review by rosemarymarano — COMMENTED** [review]
Few edits/suggestions

File: content/en/docs/troubleshooting/diagnostictools/analysisadvice/IngressInvalidShape.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: Ingress Controller Invalid Shape
+linkTitle: Ingress Controller Invalid Shape
+description: Analysis detected an invalid shape for Oracle Cloud Infrastructure Load Balancer
```
[the remark this answers is not in the document]
Comment (bhatpmk) [reply content/en/docs/troubleshooting/diagnostictools/analysisadvice/IngressInvalidShape.md]: Done

File: content/en/docs/troubleshooting/diagnostictools/analysisadvice/IngressInvalidShape.md (hunk 1 above)
Comment (bhatpmk) [reply content/en/docs/troubleshooting/diagnostictools/analysisadvice/IngressInvalidShape.md]: Removed the link from Related Information and updated the wording under "Steps", as suggested.

**Review by rosemarymarano — APPROVED** [review]
Perfect, thank you.

**Comment by bhatpmk** [comment]
VZ-6151

**Pull request merged**